### PR TITLE
[MU4] Notation stabilization

### PR DESCRIPTION
**Pull request by RomanPudashkin**

### CODE REVIEW

File: src/framework/uicomponents/qml/MuseScore/UiComponents/StyledScrollBar.qml [hunk 1]
```diff
@@ -6,23 +6,22 @@ import MuseScore.Ui 1.0
 ScrollBar {
     id: root
 
-    width: orientation === Qt.Vertical ? 10 : 0
-    height: orientation === Qt.Horizontal ? 10 : 0
+    property alias color: handle.color
+    property alias border: handle.border
 
-    visible: size !== 0
+    width: orientation === Qt.Vertical ? 10: 0
```
Comment (Eism) [inline src/framework/uicomponents/qml/MuseScore/UiComponents/StyledScrollBar.qml:12]: you missed space

File: src/commonscene/qml/MuseScore/CommonScene/TempoView.qml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.12
```
Comment (Eism) [inline src/commonscene/qml/MuseScore/CommonScene/TempoView.qml:2]: 1.15

File: src/notation/qml/MuseScore/NotationScene/qmldir [hunk 3]
```diff
@@ -9,3 +9,4 @@ ConcertPitchControl 1.0 ConcertPitchControl.qml
 NoteInputBar 1.0 NoteInputBar.qml
 NoteInputBarCustomizationDialog 1.0 NoteInputBarCustomizationDialog.qml
 UndoRedoToolBar 1.0 UndoRedoToolBar.qml
+TempoView 1.0 TempoView.qml
```
Comment (Eism) [inline src/notation/qml/MuseScore/NotationScene/qmldir:12]: why you are exporting this component from two modules(CommonScene, Notation)?

File: src/playback/qml/MuseScore/Playback/internal/MeasureAndBeatFields.qml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+RowLayout {
+    id: root
+
+    property alias measureNumber: measureNumberField.value
+    property alias maxMeasureNumber: measureNumberField.maxValue
+
+    property alias beatNumber: beatNumberField.value
+    property alias maxBeatNumber: beatNumberField.maxValue
+
+    property var font
```
Comment (Eism) [inline src/playback/qml/MuseScore/Playback/internal/MeasureAndBeatFields.qml:16]: please, add default value

**Review by Eism — APPROVED** [review]

File: src/playback/qml/MuseScore/Playback/internal/MeasureAndBeatFields.qml (hunk 4 above)
Comment (RomanPudashkin) [reply src/playback/qml/MuseScore/Playback/internal/MeasureAndBeatFields.qml]: fixed

File: src/notation/qml/MuseScore/NotationScene/qmldir (hunk 3 above)
Comment (RomanPudashkin) [reply src/notation/qml/MuseScore/NotationScene/qmldir]: removed

File: src/commonscene/qml/MuseScore/CommonScene/TempoView.qml (hunk 2 above)
Comment (RomanPudashkin) [reply src/commonscene/qml/MuseScore/CommonScene/TempoView.qml]: fixed

File: src/framework/uicomponents/qml/MuseScore/UiComponents/StyledScrollBar.qml (hunk 1 above)
Comment (RomanPudashkin) [reply src/framework/uicomponents/qml/MuseScore/UiComponents/StyledScrollBar.qml]: fixed

**Pull request merged**